src/styles/main.scss

```diff
@@ -1,3 +1,10 @@
+@import "variables";
+@import "calendar";
+
 body {
+  font-size: $font-size;
+  font-family: $font;
   margin: 0;
+  display: flex;
+  justify-content: center;
```

Comment:
In order to center the calendar properly, you need to add align-items center and set height to 100vh